app/services/db/monthly_sp_auth_count/unique_monthly_auth_counts_by_partner.rb

```diff
@@ -93,23 +93,51 @@ def call(key:, issuers:, start_date:, end_date:)
       # @return [Array<String>]
       def build_queries(issuers:, months:)
         months.map do |month_range|
+          today = Date.today
           params = {
             range_start: month_range.begin,
             range_end: month_range.end,
             year_month: month_range.begin.strftime('%Y%m'),
             issuers: issuers,
+            one_years_ago: today - 365,
+            two_years_ago: today - 2 * 365,
+            three_years_ago: today - 3 * 365,
+            four_years_ago: today - 4 * 365,
+            five_years_ago: today - 5 * 365,
           }.transform_values { |value| quote(value) }
 
+
+
           format(<<~SQL, params)
             SELECT
               sp_return_logs.user_id
             , %{year_month} AS year_month
             , COUNT(sp_return_logs.id) AS auth_count
             , sp_return_logs.ial
+            , SUM (
+              CASE WHEN sp_return_logs.profile_verified_at IS BETWEEN %{now) AND %{one_years_ago}
+              THEN 1 ELSE 0 END) as partner_ial2_new_unique_users_year1
+            , SUM (
+              CASE WHEN sp_return_logs.profile_verified_at IS BETWEEN %{one_years_ago) AND %{two_years_ago}
+              THEN 1 ELSE 0 END) as partner_ial2_new_unique_users_year2
+            , SUM (
+              CASE WHEN sp_return_logs.profile_verified_at IS BETWEEN %{two_years_ago) AND %{three_years_ago}
+              THEN 1 ELSE 0 END) as partner_ial2_new_unique_users_year3
+            , SUM (
+              CASE WHEN sp_return_logs.profile_verified_at IS BETWEEN %{three_years_ago) AND %{four_years_ago}
+              THEN 1 ELSE 0 END) as partner_ial2_new_unique_users_year4
+            , SUM (
+              CASE WHEN sp_return_logs.profile_verified_at IS BETWEEN %{four_years_ago) AND %{five_years_ago}
+              THEN 1 ELSE 0 END) as partner_ial2_new_unique_users_year5
+            , SUM (
+              CASE WHEN sp_return_logs.profile_verified_at > %{five_years_ago} 
+              THEN 1 ELSE 0 END) as partner_ial2_new_unique_users_year_greater_than_5
```

Comment:
technically [according to the doc](https://docs.google.com/document/d/1_2dg9AhPKsQKKWNLXPn3x1bDHWOS3SNDdSMTePfW5ho/edit) users must reproof every 5 years
> IDV/Proofed users must reproof every 5 years
 so "greater than 5" may not be strictly needed, but I am guessing things may always slip through so it seems OK to add here

Reply:
Correct, Should we consider adding this value in the main report?

Reply:
Let's add it for now, and make sure to solicit feedback on it from our billing partners

Reply:
Sounds good, will be good to get feedback on first pass to stakeholders